mne/viz/tests/test_topomap.py

```diff
@@ -306,6 +306,10 @@ def patch():
     assert_array_equal(_find_peaks(evoked, 10), evoked.times[[1, 95]])
     assert_array_equal(_find_peaks(evoked, 1), evoked.times[95])
 
+    # Test excluding bads channels
+    evoked_grad.info['bads'] += [evoked_grad.info['ch_names'][0]]
+    evoked_grad.plot_topomap(ch_type='grad', times=[0])
```

Comment:
can you check here that evoked_grad.info['bads'] did not change?

Reply:
Done